util-taglib/src/com/liferay/taglib/ui/InputPermissionsTag.java

```diff
@@ -36,18 +36,20 @@ public static String doTag(
 
 		return doTag(
 			_PAGE, formName, modelName, permissionPropagationCheckboxLabel,
-			false, pageContext);
+			false, false, pageContext);
 	}
 
 	public static String doTag(
 			String page, String formName, String modelName,
-			String permissionPropagationCheckboxLabel, boolean reverse,
-			PageContext pageContext)
+			String permissionPropagationCheckboxLabel, boolean defaultValue,
+			boolean reverse, PageContext pageContext)
```

Comment:
The parameters should be sorted alphabetically.

Reply:
Fixed it. Thanks